### ISSUE TITLE
Latest release breaks File/Open File on remote X shell

### ISSUE THREAD

**Issue by russell-taylor** [microsoft/vscode#178821]
Type: <b>Bug</b>

I just got notification of an update today. Before that, things were working properly.

I am running Visual Studio Code using a remote SSH tunnel (ssh -X). When I attempt to open a file using the file menu now, it pops up the file-open dialog on the system console rather than the remote X session. This happens even when I've logged out of the console.

I used to be able to open files, with the dialog box popping up as expected on the remote display client.

VS Code version: Code 1.77.0 (7f329fe6c66b0f86ae1574c2911b681ad5a45d63, 2023-03-29T10:05:35.165Z)
OS version: Linux x64 4.15.0-208-generic snap
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 x 3775)|
|GPU Status|2d_canvas: unavailable_software<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>opengl: disabled_off<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software<br>webgpu: disabled_off|
|Load (avg)|1, 6, 9|
|Memory (System)|31.36GB (28.66GB free)|
|Process Argv|--no-sandbox --crash-reporter-id e4ef60d2-4c7c-406f-9c21-d8c5ac60e87a|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|undefined|
|XDG_CURRENT_DESKTOP|undefined|
|XDG_SESSION_DESKTOP|undefined|
|XDG_SESSION_TYPE|undefined|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
doxdocgen|csc|1.4.0
better-cpp-syntax|jef|1.17.2
vscode-docker|ms-|1.24.0
isort|ms-|2022.8.0
python|ms-|2023.6.0
vscode-pylance|ms-|2023.3.40
jupyter|ms-|2023.3.1000892223
jupyter-keymap|ms-|1.1.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.288.0
cmake-tools|ms-|1.13.45
cpptools|ms-|1.14.5
cpptools-extension-pack|ms-|1.3.0
cmake|twx|0.0.17
volar|Vue|1.2.0
debug|web|0.26.1

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vscaat:30438848
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30610679
nodejswelcome1cf:30587006
2e4cg342:30602488
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
6233i204:30672705
vsccsb:30677849
pythonb192:30669360
defaultazurewalk:30687958
pythonms35:30701012

```

</details>

**Comment by russell-taylor** [comment]
Host/server is running Ubuntu 18.04. Client/remote is running Windows 11 using MoboXterm.

**Comment by russell-taylor** [comment]
@deepak1556 This is still happening with release 1.77.1 and it happens with Ubuntu 22.04 using 'ssh -X' as the client. So it is not client-dependent. It also happens on an Ubuntu 22.04 server connected using 'ssh -X' from an Ubuntu 18.04 client. So it is not server dependent.

Should be easy to reproduce.

**Comment by russell-taylor** [comment]
This is still happening in 1.77.2. I've had to revert three times now to keep being able to open files.

**Comment by russell-taylor** [comment]
Still happening in 1.77.3.

**Comment by russell-taylor** [comment]
This is fixed in 1.78.1
